man/man1/arcstat.1

```diff
@@ -0,0 +1,482 @@
+\fBphit \fR
+.ad
+.RS 14n
+Prefetch hits per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBpmis \fR
+.ad
+.RS 14n
+Prefetch misses per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBread \fR
+.ad
+.RS 14n
+Total ARC accesses per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBtime \fR
+.ad
+.RS 14n
+Time
+.RE
+
+.sp
+.ne 2
+.na
+\fBarcsz \fR
+.ad
+.RS 14n
+ARC size
+.RE
+
+.sp
+.ne 2
+.na
+\fBdread \fR
+.ad
+.RS 14n
+Demand data accesses per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBeskip \fR
+.ad
+.RS 14n
+evict_skip per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBmiss% \fR
+.ad
+.RS 14n
+ARC miss percentage
+.RE
+
+.sp
+.ne 2
+.na
+\fBmread \fR
+.ad
+.RS 14n
+Metadata accesses per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBpread \fR
+.ad
+.RS 14n
+Prefetch accesses per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBl2hit% \fR
+.ad
+.RS 14n
+L2ARC access hit percentage
+.RE
+
+.sp
+.ne 2
+.na
+\fBl2hits \fR
+.ad
+.RS 14n
+L2ARC hits per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBl2miss \fR
+.ad
+.RS 14n
+L2ARC misses per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBl2read \fR
+.ad
+.RS 14n
+Total L2ARC accesses per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBl2size \fR
+.ad
+.RS 14n
+Size of the L2ARC
+.RE
+
+.sp
+.ne 2
+.na
+\fBmtxmis \fR
+.ad
+.RS 14n
+mutex_miss per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBl2bytes \fR
+.ad
+.RS 14n
+Bytes read per second from the L2ARC
+.RE
+
+.sp
+.ne 2
+.na
+\fBl2miss% \fR
+.ad
+.RS 14n
+L2ARC access miss percentage
+.RE
+
+.sp
+.ne 2
+.na
+\fBl2asize \fR
+.ad
+.RS 14n
+Actual (compressed) size of the L2ARC
+.RE
+
+.sp
+.ne 2
+.na
+\fBgrow \fR
+.ad
+.RS 14n
+ARC grow disabled
+.RE
+
+.sp
+.ne 2
+.na
+\fBneed \fR
+.ad
+.RS 14n
+ARC reclaim needed
+.RE
+
+.sp
+.ne 2
+.na
+\fBfree \fR
+.ad
+.RS 14n
+ARC free memory
+.RE
+.\"
+
+.SH OPTIONS
+.LP
+The following options are supported:
+
+.sp
+.ne 2
+.na
+\fB\fB-f\fR\fR
+.ad
+.RS 12n
+Display only specific fields. See \fBDESCRIPTION\fR for supported statistics.
+.RE
+
+.sp
+.ne 2
+.na
+\fB\fB-h\fR\fR
+.ad
+.RS 12n
+Display help message.
+.RE
+
+.sp
+.ne 2
+.na
+\fB\fB-o\fR\fR
+.ad
+.RS 12n
+Report statistics to a file instead of the standard output.
+.RE
+
+.sp
+.ne 2
+.na
+\fB\fB-s\fR\fR
+.ad
+.RS 12n
+Display data with a specified separator (default: 2 spaces).
+.RE
+
+.sp
+.ne 2
+.na
+\fB\fB-x\fR\fR
+.ad
+.RS 12n
+Print extended stats (same as -f time,mfu,mru,mfug,mrug,eskip,mtxmis,dread,pread,read).
+.RE
+
+.sp
+.ne 2
+.na
+\fB\fB-v\fR\fR
+.ad
+.RS 12n
+Show field headers and definitions
+.RE
+
+.SH OPERANDS
+.LP
+The following operands are supported:
+.sp
+.ne 2
+.na
+\fB\fIcount\fR\fR
+.ad
+.RS 12n
+Display only \fIcount\fR reports.
+.RE
+
+.sp
+.ne 2
+.na
+\fB\fIinterval\fR\fR
+.ad
+.RS 12n
+Specify the sampling interval in seconds.
+.RE
+
+.SH AUTHORS
+.LP
+arcstat was originally written in Perl by Neelakanth Nadgir and supported only ZFS ARC statistics.
+Mike Harsch updated it to support L2ARC statistics.
+John Hixson ported it to Python for FreeNAS over some beer, after which many individuals from the FreeNAS community continued to maintain and improve it.
```

Comment:
How about "many individuals from the OpenZFS community" instead to recognize contributors regardless of platform.  Aside from that this looks good to me.

When updating I'd just suggest removing the commit comment which disables the CI and updating the PR one last time.  This is just a documentation change, but it does touch a Makefile so I don't mind being on the safe side.